sync/rest/document-permissions/delete-permission/delete-permission.3.x.js

```diff
@@ -5,7 +5,7 @@ var Twilio = require('twilio').Twilio;
 var client = new Twilio(accountSid, authToken);
 var service = client.sync.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX');
 
-service.documents('MyFirstDocument')
+service.documents('ETXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
```

Comment:
Note that using unique name is intentional here.
It is easier to open the document by UniqueName instead of SID, since applications cannot hardcode the latter in their business logic.